src/content/desci/index.md

```diff
@@ -63,9 +63,9 @@ A comparison to traditional science
    </td>
   </tr>
   <tr>
-   <td>You raise funding from a global community.
+   <td>You raise funding from a decentralized community.
```

Comment:
Not everyone wants to get funding from a decentralized community. The good thing is that DeSci does not require that you do. I think this should highlight choice and not an absolute.